src/features/course/use-comments.ts

```diff
@@ -0,0 +1,172 @@
+import { useState } from "react";
+import type { CommentProps } from "./comment";
+
+type ReactionType = "fire" | "heart" | "lightbulb" | "thumbsUp";
+
+const mockComments: CommentProps[] = [
+  {
+    id: "1",
+    rating: "9/10",
+    content:
+      "Mentoring przerósł moje oczekiwania! Zaczynałem od zera, a teraz pracuję jako junior frontend developer. Mentor w przystępny sposób tłumaczył nawet najtrudniejsze zagadnienia, a do tego miał ogromną cierpliwość. Dzięki jego wsparciu nie tylko nauczyłam się programować, ale też uwierzyłam w siebie!",
+    author: {
+      name: "Anna",
+      avatar: "/adrian-284.webp",
+      fallback: "A",
+    },
+    reactions: {
+      fire: 2,
+      heart: 19,
+      lightbulb: 4,
+      thumbsUp: 4,
+    },
+    replies: [
+      {
+        id: "1-1",
+        content: "Zgadzam się!",
+        author: {
+          name: "Tomek",
+          avatar: "/adrian-284.webp",
+          fallback: "T",
+        },
+        reactions: {
+          fire: 2,
+          heart: 19,
+          lightbulb: 4,
+          thumbsUp: 4,
+        },
+      },
+      {
+        id: "1-2",
+        content: "Bardzo pomocna opinia, dziękuję za podzielenie się!",
```

Comment:
This widget is making content jumping.

I think maybe we can do it like that
1. Make it absolute while hover
2. Then no contnet jumping.

<img width="1147" height="335" alt="image" src="https://github.com/user-attachments/assets/a7c23487-aa1a-496e-9040-bd54c39cdd86" />